FirebaseDataConnect/README.md

```diff
@@ -1,19 +1,126 @@
+
+### 1. Create a new Data Connect service and Cloud SQL instance.
+* Go to Firebase Console and select Firebase Data Connect from LeftNav bar to create a new Data Connect service and a Cloud SQL instance. You have to be in Blaze plan and you can view the details of pricing at https://firebase.google.com/pricing.
+* Select us-central1 region if you want to try out vector search with Data Connect later.
+* Wait for the Cloud SQL instance to be provisioned, you can view and manage the instance at the [Cloud console](https://pantheon.corp.google.com/sql).
+
+
+### 2. Set up [Firebase CLI](https://firebase.devsite.corp.google.com/docs/cli)
+
+* If you already have CLI, make sure you always update to the latest version
+```
+npm install -g firebase-tools
+```
+
+### 3. Set up VSCode
+You will need VS Code and its Firebase extension (VS Code extension) to automatically generate Swift code for your queries.
+* Install VS Code
+* Download the [extension](https://firebasestorage.googleapis.com/v0/b/firemat-preview-drop/o/vsix%2Ffirebase-vscode-latest.vsix?alt=media) and drag it into the "Extensions" Left Nav bar for installation.
+* Create a fdc folder
+```
+mkdir fdc
+```
+* Open VS Code from FDC folder
+* Select the Firebase icon on the left and login
+* Click on "Run firebase init" button
+
+* Select Data Connect
+* Select the project, service and database ID you setup on the console
+* Enter to select the default connector ID and complete the rest of the process
+
+### 4. Create a schema and generate some dummy data
+* In the schema.gql file, uncomment the schema
+```
+type User @table(key: "uid") {
+   uid: String!
+   name: String!
+   address: String!
+}
+```
+* On top of the schema User, an "Add data" button start showing up, click on it to generate a User_insert.gql file
+* Fill out the fields and click on Run button to run the query to add a user dummy data for testing
+
+### 5. Deploy your schema
+* Select the Firebase icon on the left and Click on the "Deploy all" button to deploy all the schema and operations to backend.
+* You can now see your schemas on the Firebase Console.
+
+### 6. Set up a mutation 
+In the mutations.gql file, uncomment the "CreateUser" query.
+* In the CONFIGURATION -> VARIABLES, enter
+```
+{
+  "name" : "dummy_name",
+  "address" : "dummy_address"
+}
+```
+* In the CONFIGURATION -> AUTHENTICATION, select Run as "Authenticated".
+* Click on the "Run" button above the query.
+* You should see your dummy data is added.
+* Select the Firebase icon on the left and Click on the "Deploy all" button to deplouy all the schema and operations to backend.
+
+### 7. Set up generated SDK location
+In the connector.yaml file, add the following code to enable swift code to be generated.
+
+```
+  swiftSdk:
+     outputDir: "../swift-generated/"
+     package: "User"
+     coreSdkPackageLocation: "file:///Users/[YOURUSERID]/github/firebase-ios-sdk"
```

Comment:
The team is schedule a new version of cli release today, so the developer don't need to include this line `coreSdkPackageLocation:` in yaml file.

Reply:
Currently the default option doesn't do anything setting the path, we have a meeting this afternoon to discuss so I think we still need this.